docs/api/browser-window.md

```diff
@@ -327,11 +327,11 @@ Returns an array that contains window's width and height.
 * `width` Integer
 * `height` Integer
 
-Resizes the window's client area (e.g. the web page) to `width` and `height`.
+Resizes the renderer's area (i.e. the web page) to `width` and `height`.
```

Comment:
The `client area` is a desktop development term, and should not be replaced with `renderer`.